resources/views/auth/login.blade.php

```diff
@@ -1,48 +1,47 @@
 
-        <form method="POST" action="{{ route('login') }}">
-            @csrf
+                   @if (session('status'))
+                       <div class="mb-4 font-medium text-sm text-green-600">
+                           {{ session('status') }}
+                       </div>
+                   @endif
 
-            <div>
-                <x-jet-label for="email" value="{{ __('Email') }}" />
-                <x-jet-input id="email" class="block mt-1 w-full" type="email" name="email" :value="old('email')" required autofocus />
-            </div>
+                    <form method="POST" class="" action="{{ route('login') }}">
+                      @csrf
+                        <div class="mb-5">
+                            <label for="email" class="block mb-2 text-sm font-medium text-gray-600">Email</label>
 
-            <div class="mt-4">
-                <x-jet-label for="password" value="{{ __('Password') }}" />
-                <x-jet-input id="password" class="block mt-1 w-full" type="password" name="password" required autocomplete="current-password" />
-            </div>
+                            <input type="text" placeholder="Enter a valid email" name="email" class="block w-full p-3 rounded bg-gray-200 border border-transparent focus:outline-none">
+                        </div>
 
-            <div class="block mt-4">
-                <label for="remember_me" class="flex items-center">
-                    <x-jet-checkbox id="remember_me" name="remember" />
-                    <span class="ml-2 text-sm text-gray-600">{{ __('Remember me') }}</span>
-                </label>
-            </div>
+                        <div class="mb-5">
+                            <label for="password" class="block mb-2 text-sm font-medium text-gray-600">Password</label>
+
+                            <input type="password" placeholder="Enter account password" name="password" class="block w-full p-3 rounded bg-gray-200 border border-transparent focus:outline-none">
+                        </div>
+
+                        <button class="w-full p-3 mt-1 bg-indigo-600 text-white rounded shadow">Login</button>
+                    </form>
+                </div>
 
-            <div class="flex items-center justify-end mt-4">
-                @if (Route::has('password.request'))
-                    <a class="underline text-sm text-gray-600 hover:text-gray-900" href="{{ route('password.request') }}">
-                        {{ __('Forgot your password?') }}
-                    </a>
-                @endif
+                <div class="flex justify-between p-8 text-sm border-t border-gray-300 bg-gray-100">
+                    <a href="{{ route('register') }}" class="font-medium text-indigo-500">Create account</a>
 
-                <x-jet-button class="ml-4">
-                    {{ __('Log in') }}
-                </x-jet-button>
+                    <a href="#" class="text-gray-600">Forgot password?</a>
```

Comment:
What happened to the forgot password ?

Reply:
The link has been added but will have to look at the email sending functionality later